JS.md

```diff
@@ -288,10 +290,10 @@ For each [`import`](https://github.com/WebAssembly/spec/blob/master/interpreter/
 1. Let `v` be the value of performing [`Get`](http://tc39.github.io/ecma262/#sec-get-o-p)(`o`, [`i.item_name`](https://github.com/WebAssembly/spec/blob/master/interpreter/spec/ast.ml#L171))
```

Comment:
The omission of the above `TypeError` ("If `Type(o)` is not Object") is on purpose, right? It kinda happens before the constructor is called I guess?

Same question for "If the `importObject` parameter is not `undefined` and `Type(importObject)` is not Object".